client/reader/discover/index.node.js

```diff
@@ -1,15 +1,32 @@
-import { getAnyLanguageRouteParam } from '@automattic/i18n-utils';
+import {
```

Comment:
I understand this file is for logged-out, server-side rendering, specifically for SEO. See https://github.com/Automattic/wp-calypso/pull/94957